rest/src/commonMain/kotlin/json/request/GuildRequests.kt

```diff
@@ -244,6 +244,8 @@ public data class GuildModifyRequest(
     @SerialName("preferred_locale")
     val preferredLocale: Optional<String?> = Optional.Missing(),
     val features: Optional<Set<GuildFeature>> = Optional.Missing(),
+    @SerialName("safety_alerts_channel_id")
+    val safetyAlertsChannelId: OptionalSnowflake? = OptionalSnowflake.Missing,
```

Comment:
You set it as optional here, but as a required nullable in the guild data object?

Please refer to my comment above

Reply:
yeah, this is the modify request:
missing means leave it as is, null means reset it

https://discord.com/developers/docs/resources/guild#modify-guild-json-params says it's optional and nullable (all parameters are optional as described in the info box above)